samples/vite/src/Agent.tsx

```diff
@@ -257,11 +248,29 @@ const Agent = () => {
           Title with content
         </Title>
       </article>
-
       <article>
         <Title heading="h3">Deprecated stuff</Title>
         <SelectBase />
       </article>
+      <BooleanModal
+        ref={modalRef}
+        id="boolean-modal"
+        title="Boolean Modal title"
+        onCancel={() => modalRef.current?.close()}
+        onSubmit={() => modalRef.current?.close()}
+        cancelTitle="Annuler"
+        submitTitle="Valider"
+        size="lg"
+      >
+        <p>
+          Reprehenderit sit quis aute nisi consequat consequat mollit. Commodo
+          in aliquip consectetur nulla sit anim. Pariatur minim commodo enim ea
+          eu laborum culpa laboris. Labore labore irure ipsum consequat enim
+          officia anim ipsum aliqua excepteur qui sint. Duis sint do culpa
+          adipisicing dolor adipisicing ea dolore aute nisi quis ullamco aliquip
+          occaecat. Aute ut mollit amet.
+        </p>
+      </BooleanModal>{" "}
```

Comment:
Un petit ajout involontaire d'eslint ou de prettier 😅 
```suggestion
      </BooleanModal>
```

Reply:
mince, je corrige